.github/workflows/build_truss_push.yaml

```diff
@@ -1,23 +1,46 @@
 ---
-name: Build Truss Image Push
-'on':
+name: Build and push Truss image
+on:
   push:
     branches:
     - truss
+
 jobs:
   docker_build:
     runs-on: ubuntu-latest
     steps:
-    - name: Check out GitHub Repo
+    -
+      name: Check out GitHub Repo
```

Comment:
this workflow is slightly different to the PR version; I could have reused the `build_and_push_image` workflow and edited slightly, but since the Truss branch will be disappearing soon, it wasn't worth it.